modules/integration/test-ui-sp-editor/src/main/java/org/wso2/sp/selenium/components/actionObjects/rootActionObject.java

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.sp.selenium.components.actionObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+
+public class rootActionObject {
+
```

Comment:
shl we remove such new lines here and elsewhere

Reply:
Fixed in 8ba1accb6be44e5276ff80077d248840ab32c3be